src/components/Heading/README.md

```diff
@@ -1,36 +1,298 @@
 # Heading
 
+## Themed
+
```

Comment:
I think the base component documentation should simply be the default options for that component. Theming demos are more appropriate in the the theme component documentation.

Reply:
To quote myself from [another comment](https://github.com/square/maker/pull/216#issuecomment-1028356473), just for visibility's sake:

> I don't think it's a good idea to cram demos of every component into the MTheme docs because it will become a kitchen-sink-style page where it'll be hard for people to find what they're looking for and it'll be hard for us as devs to maintain it. It's much simpler and easier, for both consumers and maintainers of Maker, if the themed demos go into all of the individual component's docs pages.

Reply:
It's far more effective to keep theming and documentation about theming separate from each component. We can create any number of labs from the theme documentation the demonstrates how it all works. The theme is exposing controls for color, fonts, and shape and we already have a "kitchen sink" demo. This is extremely common to have separate theming docs that don't bleed into or overtake the component documentation:

- https://material.io/design/material-theming/overview.html#material-theming
- https://m3.material.io/foundations/customization
- https://www.carbondesignsystem.com/guidelines/themes/overview/
- https://www.carbondesignsystem.com/guidelines/typography/overview/

When you're looking at a specific component the ideal scenario is a clear and static version of that component. What are the common variants / props a person would customize. Make it easy to copy and paste a single use-case. 

We're trying to build Maker so that consumers don't even need to think about theming, so therefore the component documentation should mimic how we believe most people to understand how that component works out-of-the-box. We can link to Theming documentation if it's relevant, but you shouldn't have to learn about Theming to understand a component.